site/en/tutorials/load_data/csv.ipynb

```diff
@@ -244,6 +244,7 @@
         "      na_value=\"?\",\n",
         "      num_epochs=1,\n",
         "      ignore_errors=True, \n",
+        "      shuffle=False, \n",
```

Comment:
Can you make it a parameter instead?

So for raw_train_data it will be `get_dataset(train_file_path, shuffle=True)`

raw_test_data = get_dataset(test_file_path, shuffle=False)